sympy/printing/mathml.py

```diff
@@ -584,14 +584,19 @@ def _print_Add(self, expr, order=None):
 
     def _print_MatrixBase(self, m):
         brac = self.dom.createElement('mfenced')
```

Comment:
You can move this line and the next three to after `return table` now that `brac` does not have to be defined before return in case of no delimiter.